pkg/operator/ceohelpers/util.go

```diff
@@ -0,0 +1,54 @@
+package ceohelpers
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+
+	"github.com/ghodss/yaml"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/klog/v2"
+)
+
+const (
+	clusterConfigName      = "cluster-config-v1"
+	clusterConfigKey       = "install-config"
+	clusterConfigNamespace = "kube-system"
+)
+
+type replicaCountDecoder struct {
+	ControlPlane struct {
+		Replicas string `yaml:"replicas,omitempty"`
+	} `yaml:"controlPlane,omitempty"`
+}
+
+// GetMastersReplicaCount get number of expected masters statically defined by the controlPlane replicas in the install-config.
+func GetMastersReplicaCount(ctx context.Context, kubeClient kubernetes.Interface) (int, error) {
+	klog.Infof("Getting number of expected masters from %s", clusterConfigName)
+	clusterConfig, err := kubeClient.CoreV1().ConfigMaps(clusterConfigNamespace).Get(ctx, clusterConfigName, metav1.GetOptions{})
```

Comment:
this is not good. We cannot have a quorum read for such a little number. Move this func where it is needed and tend to use a lister, if it is called more than once, e.g. in the controller sync func.